spec/features/idv/doc_auth/welcome_step_spec.rb

```diff
@@ -29,6 +29,37 @@ def expect_doc_auth_first_step
     it_behaves_like 'ial2 consent without js'
   end
 
+  context 'skipping upload step', :js, driver: :headless_chrome_mobile do
+    let(:fake_analytics) { FakeAnalytics.new }
+
+    before do
+      allow_any_instance_of(ApplicationController).
+        to receive(:analytics).and_return(fake_analytics)
+
+      sign_in_and_2fa_user
+      complete_doc_auth_steps_before_welcome_step
+      find('label', :text => /^By checking this box/).click
```

Comment:
style nit, we use the new-style hash:

```suggestion
      find('label', text: /^By checking this box/).click
```

Reply:
> style nit, we use the new-style hash:

Whoops. I'd adapted this from a sample snippet and must have overlooked the hash style.

On that note, would it make sense to enable the [HashSyntax cop](https://www.rubydoc.info/gems/rubocop/0.46.0/RuboCop/Cop/Style/HashSyntax)? 🤔